server/api_test.go

```diff
@@ -551,17 +551,11 @@ func TestHandler(t *testing.T) {
 		mockConfiguration.EXPECT().GetConfig().Return(&model.Config{}).Times(1)
 		mockChannel.EXPECT().Get(channelID).Return(&model.Channel{Id: channelID, DeleteAt: 1}, nil).Times(1)
 		mockUser.EXPECT().HasPermissionToChannel(userID, channelID, model.PermissionReadChannel).Return(true).Times(1)
-		mockUser.EXPECT().HasPermissionTo(userID, model.PermissionManageSystem).Return(false).Times(1)
 		mockConfiguration.EXPECT().GetConfig().Return(&model.Config{
 			TeamSettings: model.TeamSettings{
 				ExperimentalViewArchivedChannels: &falseValue,
 			},
 		}).Times(2)
-		mockConfiguration.EXPECT().GetConfig().Return(&model.Config{
```

Comment:
Not sure why I had to fix all these tests, they pass in master but they shouldn't